### ISSUE TITLE
vehicles accelerate abnormally  when I enter and leave my apartment

### ISSUE THREAD

**Issue by Maxhero54** [MaDHouSe79/mh-drifting#2]
After installing the drift kit on the vehicle, when I enter and leave my apartment, the vehicles start to drive at twice the speed.
I'm using qb-apartments (2.2.1) by Kakarot

**Comment by MaDHouSe79** [comment]
yeah i know, i don't have a fix right now, sorry, but you can disable the driftmode before you enter your apartment.

**Comment by Maxhero54** [comment]
> yeah i know, i don't have a fix right now, sorry, but you can disable the driftmode before you enter your apartment.

Thank you for the quick reply. I will close it, but cunning players want to take advantage of this situation without closing it :)

any way thank you ...
